ports/stm32/eth.c

```diff
@@ -192,9 +267,27 @@ STATIC int eth_mac_init(eth_t *self) {
             return -MP_ETIMEDOUT;
         }
     }
+    #endif
 
     // Set MII clock range
     uint32_t hclk = HAL_RCC_GetHCLKFreq();
+    #if defined(STM32H7)
+    tmpreg = ETH->MACMDIOAR;
```

Comment:
call it `cr_div` like below?

Reply:
Fixed, and removed the whole tmpreg stuff.